lib/Runtime/Base/Utf8SourceInfo.h

```diff
@@ -389,7 +406,15 @@ namespace Js
         ULONG parseFlags;
         ULONG byteCodeGenerationFlags;
 
-        Utf8SourceInfo(ISourceHolder *sourceHolder, int32 cchLength, SRCINFO const* srcInfo, DWORD_PTR secondaryHostSourceContext, ScriptContext* scriptContext, bool isLibraryCode);
+        Utf8SourceInfo(ISourceHolder *sourceHolder, int32 cchLength, SRCINFO const* srcInfo,
+            DWORD_PTR secondaryHostSourceContext, ScriptContext* scriptContext, bool isLibraryCode
+#ifdef NTBUILD
+            );
+#else
+            , Js::Var scriptSource);
```

Comment:
So many #ifdef doesn't look good, we should make it a required parameter. 

---
In reply to: [85627248](https://github.com/Microsoft/ChakraCore/pull/1830#discussion_r85627248) [](ancestors = 85627248)